### ISSUE TITLE
Column content is picked up from library, but not from Current Schematics

### ISSUE THREAD

**Issue by Tarik-T** [SchrodingersGat/KiBoM#172]
First of all, great tool. 
It's the only tool for me at the moment that provides a .csv file that I don't have to adapt to get a usable list in Excel
Thanks a lot for that!

However, the following problem:
In the library file (some .kicad_sym), description, datasheet, part number, order info, footprint, etc. are maintained (or not maintained).
In the schematic the information might be maintained differently, probably more up to date or at least more suitable for the current purposes.
My impression is that KiBOM pulls the information from the library file, which, to the extent that the more current information is in the schematic, leads to rework.  Assuming component information in the schematic is more current, I would prefer a hierarchy. 
First step, consider the details given to the symbol in the schematic, if there is nothing there, use the information from the library.
Is there a solution?

Thanks!
Tarik

**Comment by SchrodingersGat** [comment]
@Tarik-T it should be giving preference to the schematic over the base library. That is my experience from regular use of KiBoM.

Can you provide a minimum working example showing this issue?

**Comment by Tarik-T** [comment]
Hi,
Of Course
Here the schematics info
![Schema](https://user-images.githubusercontent.com/77569627/195383964-ea7bd839-69de-4561-840e-1b6354a86680.png)

Here how it looks in library editor:
![Lib](https://user-images.githubusercontent.com/77569627/195388141-3175deed-b45b-4ff7-9b5c-bba25ff123e3.png)
You might note, there is a "Schema" or a "lib" in the lines.

Result in BoM
![BoM](https://user-images.githubusercontent.com/77569627/195395906-d869480f-a97b-4f43-8d91-b558d15defbf.png)

The description does not seem to be from the schematic or the library.
My understanding of solution, there are several entries in the kicad_sch file. This is what is evaluated by KiBoM:
![3_kicad_sch](https://user-images.githubusercontent.com/77569627/195399156-e1d44ce9-a090-4606-b804-609386fc72dc.png)

That is the one I m wondering if it could make more sense:
![2_kicad_sch](https://user-images.githubusercontent.com/77569627/195399353-34f8af56-df4b-4193-9385-106abf7e386b.png)


In summary, only the "Description" and "Part" fields are affected.
The description was used incorrectly by me.
Field "Part" ... I am not sure what is right or wrong.
Anyway, nothing more to complain.
Thanks for your patience - I ll close the ticket. 

Cheers,
Tarik

**Comment by SchrodingersGat** [comment]
@Tarik-T you closed this out - did you get it working?

**Comment by Tarik-T** [comment]
@SchrodingersGat 

Yes. 
Nothing to complain any more.
Thank you for that nice piece of software!

Tarik
